src/test/HostActivationTests/DotNetBuilder.cs

```diff
@@ -47,21 +60,22 @@ public DotNetBuilder(string basePath, string builtDotnet, string name)
         /// </summary>
         /// <param name="version">Version to add</param>
         /// <remarks>
-        /// Product runtime binaries are not added. All the added mock framework will contain is a mock version of host policy.
+        /// Product runtime binaries are not added. This is a simple mock.
         /// </remarks>
-        public DotNetBuilder AddMicrosoftNETCoreAppFramework(string version)
+        public DotNetBuilder AddMicrosoftNETCoreAppFramework(string version, MockType mockType = MockType.HostPolicy)
```

Comment:
I think it would be better to have a separate method for each of the mock types.
It works better with intellisense when writing the tests (won't pick the wrong one just because I don't know about mock types).
It will also let us add additional parameters if needed (might be used for the hostpolicy version for example).